composite-actions/frontend-generic/upload-source-maps/scripts/upload-source-maps.js

```diff
@@ -17,14 +17,6 @@
       FormData = (await import("form-data")).default;
     }
```

Comment:
are we not using this anywhere?. is actions works fine?

Reply:
@Vasanth-ExtendaRetail What are we not using, `FormData`??

Reply:
> are we not using this anywhere?. is actions works fine?
If you're referring to `FormData`, yes, we are using it in the script!